homeassistant/components/ecobee/climate.py

```diff
@@ -727,6 +755,63 @@ def resume_program(self, resume_all):
         )
         self.update_without_throttle = True
 
+    def set_sensors_used_in_climate(
+        self, sensors: list[str], climate_name: str | None = None
```

Comment:
I think we should rename `climate_name` to `preset_mode`or something similar as it would be easier for the user to understand it

```suggestion
        self, sensors: list[str], preset_mode: str | None = None
```